src/app/app.element.ts

```diff
@@ -40,65 +38,72 @@ export class UmbAppElement extends UmbLitElement {
 	 */
 	@property({ type: String })
 	// TODO: get from server config
-	private backofficePath = import.meta.env.DEV ? '/' : '/umbraco';
+	backofficePath? = '/umbraco';
```

Comment:
I think we can assume it is not potentially undefined

```suggestion
	backofficePath = '/umbraco';
```